File: challenges/2-idade-em-dias/elixir/citadin/idade_em_dias.exs
defmodule Challenge do
  def idade_em_dias(dias) do
    anos = div(dias, 365)
    meses = rem(dias, 365) |> div(30)
    dias = rem(dias, 365) |> rem(30)

    "#{anos} ano(s), #{meses} mes(es), #{dias} dia(s)"
  end
end

File: challenges/2-idade-em-dias/java/camilealheiro/IdadeEmDias.java
import java.util.Scanner;

public class IdadeEmDias {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        System.out.println("Digite uma quantidade de dias: ");
        int num = sc.nextInt();

        int dias, anos, meses;

        anos = num / 365;
        meses = (num % 365) / 30;
        dias = (num % 365) % 30;

        System.out.printf("%d ano(s)\n%d mes(es)\n%d dia(s)", anos, meses, dias);
    }
}
